core/services/relay/evm/median.go

```diff
@@ -54,18 +58,26 @@ func newMedianContract(contractAddress common.Address, chain evm.Chain, specID i
 			NewRoundRequestedDB(db.DB, specID, lggr),
 			chain.Config(),
 		),
+		mercuryMode: mercuryMode,
 	}, nil
 }
 
 func (oc *medianContract) Start() error {
-	return oc.tracker.Start()
+	return oc.requestRoundTracker.Start()
 }
 
 func (oc *medianContract) Close() error {
-	return oc.tracker.Close()
+	return oc.requestRoundTracker.Close()
 }
 
 func (oc *medianContract) LatestTransmissionDetails(ctx context.Context) (ocrtypes.ConfigDigest, uint32, uint8, *big.Int, time.Time, error) {
+	if oc.mercuryMode {
+		// Bit of a hack, this must return the correct config digest at least
+		// TODO: Return the actual latest transmission details
+		// https://app.shortcut.com/chainlinklabs/story/57500/return-the-actual-latest-transmission-details
+		_, cd, err := oc.configTracker.LatestConfigDetails(ctx)
+		return cd, 0, 0, big.NewInt(0), time.Time{}, err
```

Comment:
I would say, make sure returning 0s here as latest transmission details doesn't break the OCR protocol.
But if you've already verified that, then good.

Reply:
Yes this has been checked with Lorenz and verified in production, it works fine :+1: